DOM_manipulation_and_events.md: minor text changes (#25298)
TheOdinProject/curriculum: foundations/javascript_basics/DOM_manipulation_and_events.md

</head>
~~~

> Read the fourth heading in [this MDN article](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML#applying_css_and_javascript_to_html) titled "Applying CSS and JavaScript to HTML" for more information, which also includes a link to additional script loading strategies.

Reviewer: Thanks for contributing @Yusuf219, since we're linking directly to the section, we can simplify to this:
 
```suggestion
Read the [Applying CSS and JavaScript to HTML](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML#applying_css_and_javascript_to_html) section on MDN for more information, which also includes a link to additional script loading strategies.
```
